pkg/secrets/clients/k8s/k8s_secrets_client.go

```diff
@@ -32,15 +28,21 @@ func PatchK8sSecret(namespace string, secretName string, stringDataEntriesMap ma
 	// get K8s client object
 	kubeClient, _ := configK8sClient()
 
-	stringDataEntry, err := generateStringDataEntry(stringDataEntriesMap)
+	log.Info(messages.CSPFK006I, secretName, namespace)
+	k8sSecret, err := kubeClient.CoreV1().Secrets(namespace).Get(secretName, metav1.GetOptions{})
+
 	if err != nil {
-		return log.RecordedError(messages.CSPFK024E)
+		return err
```

Comment:
I see that you removed the RecordedError. This makes sense as `CSPFK024E Failed to create stringDataEntry` does not apply anymore. However, can you please return some other error of what happened?

Also - please remove any message that is not used from the messages files (e.g `CSPFK024E`)